utils/session.c

```diff
@@ -911,10 +911,16 @@ TEST_CASE(task_symbol)
 	FILE *fp;
 
 	fp = fopen("sid-test.map", "w");
+    if(fp == NULL)
+        pr_err("cannot open %s file", "sid-test.map");
```

Comment:
Please check your editor settings - we use TAB size 8 for indentation.

Also you'd better using `TEST_NE(fp, NULL);` instead.